package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "input-plugin-email",
-  "version": "0.1.0",
+  "keyword": "punchcard-cms",
```

Comment:
Can you please add a `"version": "0.0.0"`? That way we can import non-production branches when needed.